src/modules/erc20linear.ts

```diff
@@ -0,0 +1,54 @@
+// handler methods for the erc20 linear reward module
+
+import { Address, BigInt, log } from '@graphprotocol/graph-ts'
+import {
+  ERC20LinearRewardModule as ERC20LinearRewardModuleContract,
+} from '../../generated/templates/StakingModule/ERC20LinearRewardModule'
+import { RewardsFunded } from '../../generated/templates/RewardModule/ERC20BaseRewardModule'
+import { Staked, Unstaked, Claimed } from '../../generated/templates/StakingModule/ERC20StakingModule'
+import { Pool, Token, Position, } from '../../generated/schema'
+import { integerToDecimal } from '../util/common'
+import { ZERO_BIG_INT, ZERO_BIG_DECIMAL, ONE_E_18, INITIAL_SHARES_PER_TOKEN } from '../util/constants'
+
+
+export function handleRewardsFundedLinear(event: RewardsFunded, pool: Pool, token: Token): void {
+  let contract = ERC20LinearRewardModuleContract.bind(event.address);
+
+  // update timeframe for pool
+  if (pool.start.equals(ZERO_BIG_INT)) pool.start = event.params.timestamp;
+
+  // TODO do we still want to create an object here?
+}
+
+
+export function handleStakedLinear(event: Staked, pool: Pool, position: Position): void {
+  let contract = ERC20LinearRewardModuleContract.bind(event.address);
+  // TODO anything useful here?
```

Comment:
I think its still useful to track `Position`s for `Stream` pool owners to have a view of all users in the pool

Reply:
the `Position` is still being updated currently, just not adding any underlying `Stake` objects

Reply:
Ah got it, I don't think we need anything here then

Reply:
:+1: